packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -270,6 +272,18 @@ const DatePickerInput = (props) => {
     }
   };
 
+  const handleInvalidInputChange = (val) => {
```

Comment:
`handleInvalidInputChange` is always called with value true in that case do you still need the parameter `val`

Reply:
changes done. Thanks.